README.md

```diff
@@ -1 +1,16 @@
-# data-structure-and-algorithms
\ No newline at end of file
+# Shift an Array
+
```

Comment:
Missing "<!-- Short summary or background information -->" portion of documentation